src/com/facebook/buck/apple/project_generator/ProjectGenerator.java

```diff
@@ -1503,6 +1506,27 @@ private PBXNativeTarget generateBinaryTarget(
     return target;
   }
 
+  private ImmutableList<String> getExcludesForModulesUnderTests(
```

Comment:
Nit: I'd name this `getFlagsForExcludesForModulesUnderTests` so that it's clear that it returns flags.